### DISCUSSION TITLE
Example app with Liveblocks

### DISCUSSION THREAD

**Discussion by Gyurmatag** [steven-tey/novel#230] [Q&A]
Just wondering if any one of you have built something with [LiveBlocks](https://liveblocks.io/) and Novel.sh?
Would be great to have some examples!

**Comment by yogeshbhutkar** [comment]
You can integrate LiveBlocks with Novel.sh, follow the same steps and pass all the liveblocks extensions to the NovelEditor's extensions and it works like a charm.
